### PR TITLE
Update networking options of db resources

### PR DESCRIPTION
**Pull request by paaanic**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,10 @@ description: |-
 # VKCS Provider's changelog
 
 #### v0.1.15 (unreleased)
+- Add a note to "uuid" field of resource_vkcs_db_instance.network, resource_vkcs_db_cluster.network, and resource_vkcs_db_cluster_with_shards.network that not setting a value to "uuid" is deprecated and it will be marked as required in future major release
```
Comment (ftersin) [inline CHANGELOG.md:11]: we do not mention here doc changes

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,6 +8,10 @@ description: |-
...
+- Add "subnet_id" field to resource_vkcs_db_instance.network, resource_vkcs_db_cluster.network, and resource_vkcs_db_cluster_with_shards.shard.network
+- Add a warning that is thrown when fields "network.fixed_ip_v4" and "replica_of" of resource_vkcs_db_instance are set simultaneously
```
Comment (ftersin) [inline CHANGELOG.md:13]: ditto

File: CHANGELOG.md [hunk 3]
```diff
@@ -8,6 +8,10 @@ description: |-
 # VKCS Provider's changelog
 
 #### v0.1.15 (unreleased)
+- Add a note to "uuid" field of resource_vkcs_db_instance.network, resource_vkcs_db_cluster.network, and resource_vkcs_db_cluster_with_shards.network that not setting a value to "uuid" is deprecated and it will be marked as required in future major release
+- Add "subnet_id" field to resource_vkcs_db_instance.network, resource_vkcs_db_cluster.network, and resource_vkcs_db_cluster_with_shards.shard.network
```
Comment (ftersin) [inline CHANGELOG.md:12]: not field but argument

File: vkcs/resource_vkcs_db_cluster.go [hunk 4]
```diff
@@ -176,16 +176,24 @@ func resourceDatabaseCluster() *schema.Resource {
 				Elem: &schema.Resource{
 					Schema: map[string]*schema.Schema{
 						"uuid": {
+							Type:     schema.TypeString,
+							Optional: true,
+							ForceNew: true,
+							Description: "The id of the network. Changing this creates a new cluster." +
+								"**Note** Using this field as optional is deprecated. It will be marked as required in a future major release.",
```
Comment (ftersin) [inline vkcs/resource_vkcs_db_cluster.go:183]: dinap whants to make it optional in fact. So we should say that it is really required at the moment despite it is described as optional. And in the future we remove the note

File: CHANGELOG.md (hunk 1 above)
Comment (paaanic) [reply CHANGELOG.md]: Removed

File: CHANGELOG.md (hunk 3 above)
Comment (paaanic) [reply CHANGELOG.md]: Updated

File: vkcs/resource_vkcs_db_cluster.go (hunk 4 above)
Comment (paaanic) [reply vkcs/resource_vkcs_db_cluster.go]: Updated:
https://github.com/vk-cs/terraform-provider-vkcs/blob/a9ed34cda116797e1c99ff3de54c31bd0d7a2138/vkcs/resource_vkcs_db_cluster.go#L178-L184

**Review by ftersin — APPROVED** [review]

**Pull request merged**